pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java

```diff
@@ -137,12 +137,12 @@ public RunnerBuilder ethNetworkConfig(final EthNetworkConfig ethNetworkConfig) {
     return this;
   }
 
-  public RunnerBuilder discoveryHost(final String discoveryHost) {
-    this.discoveryHost = discoveryHost;
+  public RunnerBuilder advertisedHost(final String advertisedHost) {
+    this.advertisedHost = advertisedHost;
     return this;
   }
 
-  public RunnerBuilder discoveryPort(final int listenPort) {
+  public RunnerBuilder listenPort(final int listenPort) {
```

Comment:
`listenPort` feels too generic here.  I wonder if these should be p2pAdvertisedHost and p2pListenPort?  A bigger refactoring would be to make an actual `P2pConfiguration` object to match `JsonRpcConfiguration` and friends but that's probably scope creep on what you were working on. :)

Reply:
Updated the names.  We do have a `NetworkingConfiguration` object, although it's probably due for some cleanup.